Compile error with undefined reference to `sai_metadata_get_attr_metadata'

Seeing below compile error when building sathriftv2.

export SAITHRIFTV2=y
make saithrift-build

SAI: v1.11
Thrift version 0.14.1
```
g++ -I/usr/include/sai -I. -std=c++11 -O0 -ggdb -c src/saiserver.cpp  -o obj/saiserver.o -I/usr/include/sai -I. -std=c++11 -O0 -ggdb -DBRCMSAI -I./gen-cpp -I../../inc -I../../experimental
g++ -L/usr/lib -Wl,-rpath=/usr/lib ./obj/sai_rpc_server.o ./obj/saiserver.o -o saiserver \
	   ./obj/librpcserver.a -lthrift -lpthread -lsai 
/usr/bin/ld: ./obj/sai_rpc_server.o: in function `convert_attr_thrift_to_sai(_sai_object_type_t, sai::sai_thrift_attribute_t const&, _sai_attribute_t*)':
/sonic/SAI/test/saithriftv2/../../meta/sai_rpc_frontend.cpp:132: undefined reference to `sai_metadata_get_attr_metadata'
/usr/bin/ld: ./obj/sai_rpc_server.o: in function `convert_attr_sai_to_thrift(_sai_object_type_t, _sai_attribute_t const&, sai::sai_thrift_attribute_t&)':
/sonic/SAI/test/saithriftv2/../../meta/sai_rpc_frontend.cpp:554: undefined reference to `sai_metadata_get_attr_metadata'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:106: saiserver] Error 1
make[1]: Leaving directory '/sonic/SAI/test/saithriftv2'
make: *** [Makefile:46: saithrift-build] Error 2
```

--- a/test/saithriftv2/Makefile
+++ b/test/saithriftv2/Makefile
@@ -104,7 +104,8 @@ clientlib: $(PY_SOURCES) $(SAI_PY_HEADERS)
 
 saiserver: $(ODIR)/saiserver.o $(ODIR)/librpcserver.a
 	$(CXX) $(LDFLAGS) $(ODIR)/sai_rpc_server.o $(ODIR)/saiserver.o -o $@ \
-		   $(ODIR)/librpcserver.a $(LIBS) $(SAIRPC_EXTRA_LIBS)
+		   $(ODIR)/librpcserver.a ../../meta/saimetadata.o \
+		   ../../meta/saimetadatautils.o $(LIBS) $(SAIRPC_EXTRA_LIBS)
 
 install-lib: $(ODIR)/librpcserver.a
 	$(INSTALL) -D $(ODIR)/librpcserver.a $(DESTDIR)/usr/lib/librpcserver.a

